src/orders/components/OrderTransaction/components/TransactionEvents/components/EventItem.test.tsx

```diff
@@ -0,0 +1,94 @@
+import { transactionEvent } from "@saleor/orders/fixtures";
+import Wrapper from "@test/wrapper";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import React from "react";
+import { MemoryRouter } from "react-router";
+
+import { EventItem } from "./EventItem";
+
+describe("EventItem", () => {
+  it("displays correct event data", () => {
+    render(
+      <MemoryRouter>
+        <Wrapper>
+          <EventItem
+            event={transactionEvent}
+            onHover={() => undefined}
+            hoveredPspReference={null}
+            hasCreatedBy={true}
+          />
+        </Wrapper>
+      </MemoryRouter>,
+    );
+    const row = screen.getByRole("row");
+    const intl = new Intl.DateTimeFormat("en-us", {
+      timeZoneName: "short",
+      year: "numeric",
+      month: "short",
+      day: "numeric",
+      hour: "2-digit",
+      minute: "2-digit",
+    });
+
+    expect(row).toHaveTextContent("Success"); // Transaction event
+    expect(row).toHaveTextContent("Capture");
+    expect(row).toHaveTextContent(transactionEvent.amount.amount.toString());
+    expect(row).toHaveTextContent(transactionEvent.amount.currency);
+    expect(row).toHaveTextContent(transactionEvent.pspReference);
+    expect(row).toHaveTextContent(
+      intl.format(new Date(transactionEvent.createdAt)),
```

Comment:
suggestion: IMO you should put here raw string not use extra function to calculate result